db/migrate/20240830090759_add_deprecation_support_for_polyglot.rb

```diff
@@ -0,0 +1,34 @@
+class AddDeprecationSupportForPolyglot < ActiveRecord::Migration[7.0]
+  def change
+    # add_column :polyglot_languages, :column_name, :type, ...options
+    add_column :polyglot_languages, :weight, :integer, default: 0, null: false
+    add_column :polyglot_languages, :enabled, :boolean, default: true, null: false
+
+    reversible do |dir|
+      dir.up do # Will run when migrating
+        execute <<-SQL
+          UPDATE polyglot_languages SET weight = 1 WHERE id = 6;
```

Comment:
please use `WHERE name = (something)` instead of `id`. This way is quite confusing since we generally don't really know what `id=6` or `id=554` stands for (looks quite a random number to me)